javascript-objects/main.js

```diff
@@ -0,0 +1,35 @@
+const student = {
+  firstName: 'John',
+  lastName: 'Bess',
+  age: '23',
+};
+const fullName = student.firstName + '' + student.lastName;
+console.log('student');
+console.log('fullName');
```

Comment:
These are just console logging the labels, you're not actually logging any of your variables